docs/translations-needed.md

```diff
@@ -252,16 +252,8 @@ If you add or update a translation run `grunt check-translations` to update the
 |                         | , opens subtitles settings dialog                                                   |
 |                         | , opens descriptions settings dialog                                                |
 |                         | , selected                                                                          |
-| ru.json (missing 14)    | Close Modal Dialog                                                                  |
-|                         | Descriptions                                                                        |
-|                         | descriptions off                                                                    |
-|                         | Audio Track                                                                         |
+| ru.json (missing 6 )    | Close Modal Dialog                                                                  |
 |                         | The media is encrypted and we do not have the keys to decrypt it.                   |
-|                         | Play Video                                                                          |
-|                         | Close                                                                               |
-|                         | Modal Window                                                                        |
-|                         | This is a modal window                                                              |
-|                         | This modal can be closed by pressing the Escape key or activating the close button. |
 |                         | , opens captions settings dialog                                                    |
```

Comment:
This shows up as title text when you hover to the "caption settings" menu item. To see this, you'd want to enable emulated text tracks in videojs and add captions:
```js
var player = videojs('vid', {
  html5: {
    nativeTextTracks: false
  }
});
```
Then after playing, if you open the captions menu and hover over the settings item, you can see this text.